docs/virtual_machines/tekton_tasks.md

```diff
@@ -48,15 +49,14 @@ User can use this command to enable `deployTektonTaskResources` feature gate in
 ```console
 oc patch hco kubevirt-hyperconverged  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
 ```
-or in TTO CR if TTO is deployed as a stand-alone without HCO
+or in SSP CR if SSP is deployed as a stand-alone without HCO
```

Comment:
```suggestion
or in SSP CR if SSP is deployed as a stand-alone without HCO:
```

Reply:
Same in line https://github.com/kubevirt/user-guide/pull/700/files#diff-a77db169553e543ddbc385a4516c88ce5fd572e287ff4df352928f33eafba3ddR48: 
"User can use this command to enable `deployTektonTaskResources` feature gate in HCO CR", add semicolum.